internal/cmd/commands/database/init.go

```diff
@@ -124,7 +125,13 @@ func (c *InitCommand) Flags() *base.FlagSets {
 	f.BoolVar(&base.BoolVar{
 		Name:   "skip-initial-login-role-creation",
 		Target: &c.flagSkipInitialLoginRoleCreation,
-		Usage:  "If not set, a default role allowing necessary grants for logging in will not be created as part of initialization. If set, the recovery KMS will be needed to perform any actions.",
+		Usage:  "If not set, a role providing necessary grants for logging in will not be created as part of initialization. If set, the recovery KMS will be needed to perform any actions.",
```

Comment:
The wording of this confuses me (the old wording wasn't any better). It seems to me that if the flag _is_ set a role providing the necessary grants will _not_ be created, no? This description is in contradiction to what I would expect the behavior of the flag to be given the name (if I don't set `--skip`, why should it skip?).

Reply:
Fixed